src/core/ext/xds/xds_route_config.cc

```diff
@@ -608,7 +608,7 @@ grpc_error_handle RetryPolicyParse(
     }
     retry_to_return.retry_back_off.max_interval = max;
   } else {
-    retry_to_return.retry_back_off.base_interval = Duration::Milliseconds(250);
+    retry_to_return.retry_back_off.base_interval = Duration::Milliseconds(25);
     retry_to_return.retry_back_off.max_interval = Duration::Milliseconds(250);
```

Comment:
This one needs to be changed from 250 to 25 as well.

Reply:
This one was 250 originally though?

Reply:
Doh!  You're right, nevermind. :)

Reply:
No worries... I originally set them both to 25, noticed this one was 250, and then errantly updated them both.